qupulse/hardware/dacs/alazar.py

```diff
@@ -146,3 +146,23 @@ def register_mask_for_channel(self, mask_id: str, hw_channel: int, mask_type='au
             raise NotImplementedError('Currently only can do cross buffer mask')
         self._mask_prototypes[mask_id] = (hw_channel, mask_type)
 
+    def measure_program(self, channels: Iterable[str]) -> Dict[str, List[float]]:
+        """
+        Get all measurements at once and write them in a dictionary.
+        """
+        data = {}
+        
+        scanline_data = self.__card.extractNextScanline()
+        
+        for op_name, meas_data in scanline_data.operationResults.items():
+            if op_name in channels:
+                for op in self.__armed_program.operations:
+                    # .name/.mask_name oder was anderes, dazu müsste ich einen Blick in die Downsample-Klasse werden
+                    # op.name=DS_C, op.mask_name=C (siehe test_awg-orig.py)
+                    if op.identifier == op_name:
+                        data[op.identifier] = list(meas_data.getAsVoltage(self.config.inputConfiguration[2].inputRange))
```

Comment:
getAsVoltage should returns a potentially very large numpy array. Is a list really needed here?